### ISSUE TITLE
Display extra text from bot messages

### ISSUE THREAD

**Issue by terceiro** [ltworf/localslackirc#275]
Hi. I was wondering if it's possible to display extra text from bot messages. For example these messages

![slack-context](https://user-images.githubusercontent.com/30027/112622503-e1424a80-8e09-11eb-92b8-1f724edc837d.png)

show up on IRC with no extra information, like this:

```
<bot> [GitLab]Antonio Terceiro (terceiro) opened merge request https://gitlab.com/Linaro/tuxmake/-/merge_requests/142 (!142 *Draft: build: run build commands with no network access*) in https://gitlab.com/Linaro/tuxmake (linaro / tuxmake)
<bot> [GitLab]Antonio Terceiro (terceiro) https://gitlab.com/Linaro/tuxmake/-/merge_requests/142#note_538149975 (commented on merge request !142) in https://gitlab.com/Linaro/tuxmake (linaro / tuxmake): *Draft: build: run build commands with no network access*
<bot> [GitLab]Antonio Terceiro (terceiro) https://gitlab.com/Linaro/tuxmake/-/merge_requests/142#note_538152549 (commented on merge request !142) in https://gitlab.com/Linaro/tuxmake (linaro / tuxmake): *Draft: build: run build commands with no network access*
```

The [slack API documentation](https://api.slack.com/reference/messaging/payload) makes me believe that to be possible.

**Comment by ltworf** [comment]
Could you add a print around here: https://github.com/ltworf/localslackirc/blob/master/slack.py#L826 to see what the messages from the bot look like?

If it's formatted, I guess the formatting needs to also be stripped before sending to IRC.

**Comment by terceiro** [comment]
Here are some of them:

```python
{
    "type": "message",
    "subtype": "bot_message",
    "text": "Antonio Terceiro pushed to branch <https://gitlab.com/Linaro/tuxmake/commits/master|master> of <https://gitlab.com/Linaro/tuxmake|linaro / tuxmake> (<https://gitlab.com/Linaro/tuxmake/compare/2272bba9189bc8bfd7577cb707e88a399930a347...e6186613b73b46ca6dd2324b317a7991c9fd96a0|Compare changes>)",
    "suppress_notification": False,
    "username": "GitLab",
    "bot_id": "BM8T57ULR",
    "team": "T03G6GWC6",
    "bot_profile": {
        "id": "BM8T57ULR",
        "deleted": False,
        "name": "tux",
        "updated": 1565989409,
        "app_id": "A0F7XDUAZ",
        "icons": {
            "emoji": ":penguin:",
            "image_64": "https://a.slack-edge.com/production-standard-emoji-assets/13.0/apple-large/1f427.png",
        },
        "team_id": "T03G6GWC6",
    },
    "attachments": [
        {
            "fallback": "<https://gitlab.com/Linaro/tuxmake/-/commit/786f3e94c7d8c05844e7706123d166baf0f708dc|786f3e94>: docs: document requirement for Python 3.6+ - Antonio Terceiro\n\n<https://gitlab.com/Linaro/tuxmake/-/commit/a1e37f66c861e86a4c0f302b23cb92b28b16a4bf|a1e37f66>: deb, rpm: enforce requirement on python3 &gt;= 3.6 - Antonio Terceiro\n\n<https://gitlab.com/Linaro/tuxmake/-/commit/e6186613b73b46ca6dd2324b317a7991c9fd96a0|e6186613>: Merge branch 'python3.6-required' into 'master' - Antonio Terceiro",
            "text": "<https://gitlab.com/Linaro/tuxmake/-/commit/786f3e94c7d8c05844e7706123d166baf0f708dc|786f3e94>: docs: document requirement for Python 3.6+ - Antonio Terceiro\n\n<https://gitlab.com/Linaro/tuxmake/-/commit/a1e37f66c861e86a4c0f302b23cb92b28b16a4bf|a1e37f66>: deb, rpm: enforce requirement on python3 &gt;= 3.6 - Antonio Terceiro\n\n<https://gitlab.com/Linaro/tuxmake/-/commit/e6186613b73b46ca6dd2324b317a7991c9fd96a0|e6186613>: Merge branch 'python3.6-required' into 'master' - Antonio Terceiro",
            "id": 1,
            "color": "334455",
        }
    ],
    "channel": "CMEJERRJQ",
    "event_ts": "1617024948.201400",
    "ts": "1617024948.201400",
}
{
    "type": "message",
    "subtype": "bot_message",
    "text": "Antonio Terceiro (terceiro) <https://gitlab.com/Linaro/tuxmake/-/issues/127#note_540153295|commented on issue #127> in <https://gitlab.com/Linaro/tuxmake|linaro / tuxmake>: *Can't install Debian package - failure of public key*",
    "suppress_notification": False,
    "username": "GitLab",
    "bot_id": "BM8T57ULR",
    "team": "T03G6GWC6",
    "bot_profile": {
        "id": "BM8T57ULR",
        "deleted": False,
        "name": "tux",
        "updated": 1565989409,
        "app_id": "A0F7XDUAZ",
        "icons": {
            "emoji": ":penguin:",
            "image_64": "https://a.slack-edge.com/production-standard-emoji-assets/13.0/apple-large/1f427.png",
        },
        "team_id": "T03G6GWC6",
    },
    "attachments": [
        {
            "fallback": "hi @fuegotest \n\nbased on the info you posted on #128, I tried on ubuntu 16.04 and indeed the instructions don' work there. I could make it work by dropping the `[signed-by=/usr/share/keyrings/tuxmake.asc]` and instead putting the signing key in `/etc/apt/trusted.gpg/tuxmake.gpg` after converting it from the armored to binary format. so those instructions do not work with of `apt` in ubuntu 16.04, but given ubuntu doesn't even have python 3.6, I wonder if it is worthy fixing this at all.",
            "text": "hi @fuegotest \n\nbased on the info you posted on #128, I tried on ubuntu 16.04 and indeed the instructions don' work there. I could make it work by dropping the `[signed-by=/usr/share/keyrings/tuxmake.asc]` and instead putting the signing key in `/etc/apt/trusted.gpg/tuxmake.gpg` after converting it from the armored to binary format. so those instructions do not work with of `apt` in ubuntu 16.04, but given ubuntu doesn't even have python 3.6, I wonder if it is worthy fixing this at all.",
            "id": 1,
            "color": "334455",
        }
    ],
    "channel": "CMEJERRJQ",
    "event_ts": "1617025488.201500",
    "ts": "1617025488.201500",
}
```

**Comment by ltworf** [comment]
https://github.com/ltworf/localslackirc/pull/280

Does it work with this?

**Comment by terceiro** [comment]
this has been fixed in #280
